.github/workflows/main.yml

```diff
@@ -0,0 +1,145 @@
+name: Docker Image CI
+
+on:
+  push:
+
+env:
+  NB_SERVICES: ${{ secrets.NB_SERVICES }}
+  OPTIMIZER_ORTOOLS_VERSION: ${{ secrets.OPTIMIZER_ORTOOLS_VERSION }}
```

Comment:
Do we have a way to put a default version ?

Reply:
It uses the default version specified in Dockerfile. Those variables allow you to override the values.

Reply:
These secrets, are they like hidden environment variables on github side or they are defined inside our repo?

Reply:
Secrets are defined inside your repo in the GitHub settings : `settings/secrets/actions`